portal-impl/src/com/liferay/portlet/documentlibrary/service/impl/DLFileEntryLocalServiceImpl.java

```diff
@@ -2488,6 +2488,15 @@ private void _copyFileEntryMetadata(
 		}
 	}
 
+	private void _deleteFile(
+			long companyId, long repositoryId, String name, String versionLabel)
+		throws PortalException {
+
+		DLStoreUtil.deleteFile(companyId, repositoryId, name, versionLabel);
+		DLStoreUtil.deleteFile(
```

Comment:
I'm not sure if this file exist in all the cases. so previously maybe we should check if `versionLabel+".index"` exists with a `DLStoreUtil.hasFile` ?

Reply:
It is not necessary. The `deleteFile` methods will do nothing if the files don't exist.